src/mumble/AudioPreprocessor.cpp

```diff
@@ -0,0 +1,176 @@
+// Copyright 2024 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "AudioPreprocessor.h"
+
+#include <utility>
+
+#include <speex/speex_preprocess.h>
+
+AudioPreprocessor::AudioPreprocessor() : m_handle(nullptr) {
+}
```

Comment:
`m_handle` should be assigned a default value in the header file so that the default ctor can simply be defaulted as well.